tsconfig.json

```diff
@@ -1,9 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2018",
-    "moduleResolution": "node",
+    "target": "es2022",
+    "module": "Node16",
+    "moduleResolution": "node16",
```

Comment:
It appears all of the newer Octokit libraries require this - https://github.com/octokit/action.js/blob/25f537f58b9d8ee779175a9601f0958b1b2ce6ac/README.md#L33-L37